### PR TITLE
Create controller and map all necessary information to show in frontend

### PR DESCRIPTION
**Pull request by conflicker**
Not sure if the information shown are enough. Let me know if you have other suggestions.

### CODE REVIEW

**Comment by conflicker** [comment]
@r-martins I'm not getting that error, can you try to run di:compile?

**Comment by r-martins** [comment]
Please ignore the error above and just see the last comment on pivotal ticket. ;)

**Comment by conflicker** [comment]
@r-martins If there's no issue in my previous commits, kindly review my recent one for showing error message for exceptions and validation.

File: Helper/Data.php [hunk 1]
```diff
@@ -988,4 +1009,27 @@ public function getSessionVl()
...
+    }
+
+
+    /**
+     * Validate public key
+     */
+    public function validateKey() {
+
+
+        $url = 'http://ws.ricardomartins.net.br/pspro/v6/auth/' . $this->getPagSeguroPubKey();
```
Comment (r-martins) [inline Helper/Data.php]: Please use https, and preferably have this url info in config.xml.

**Pull request merged**